network/Config.go

```diff
@@ -0,0 +1,29 @@
+package network
+
+// Config defines the network configuration.
+type Config struct {
+	Type string
+
+	// meta store
+	MetaPath string
+
+	// bridge config
+	BridgeConfig BridgeConfig
+}
+
+// BridgeConfig defines the bridge network configuration.
+type BridgeConfig struct {
+	Name        string
+	IP          string
+	FixedCIDR   string
```

Comment:
Could you add more annotations of this field?

Reply:
'fixed cidr' just literal meaning